content/Overview/get-started.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: get-started
+slug: get-started
+title: Get Started
+---
+
+import CliInstallSnippet from '../components/cli-install.mdx';
+
+Learn how to quickly get started with Configu.
+We'll use the Configu's CLI and the Configu's Saas Configuration Store to decalre, initalize, and export a simple set configuration data.
+
+### 1. Install the Configu CLI
+
+<CliInstallSnippet />
+
+### 2. Declare a Config schema
+
+```shell
+configu init --getting-started
+```
+
+### 3. Use a Config Store
+
+> Create a new organization in the [Configu SaaS Configuration Store](https://app.configu.com/).
```

Comment:
We may need to send him to a doc whos explaining about opening new organizations (keep in mind that once he will open a new org/login the first time the getting started feature will start)